pkg/reconciler/v1alpha1/revision/resources/deploy_test.go

```diff
@@ -275,8 +294,27 @@ func deployment(opts ...deploymentOption) *appsv1.Deployment {
 	return deployment
 }
 
+func revision(opts ...revisionOption) *v1alpha1.Revision {
+	revision := defaultRevision.DeepCopy()
+	for _, option := range opts {
+		option(revision)
+	}
+	return revision
+}
+
+func withContainerConcurrency(containerConcurrency int) revisionOption {
+	return func(revision *v1alpha1.Revision) {
+		revision.Spec.ContainerConcurrency = v1alpha1.RevisionContainerConcurrencyType(containerConcurrency)
+	}
+}
+
+func withoutLabels() revisionOption {
+	return func(revision *v1alpha1.Revision) {
```

Comment:
you can just make this `func withoutLabels(revision *v1alpha1.Revision) {` here and drop a bunch of `()` below.

Reply:
Fixed.